scripts/mkcloud

```diff
@@ -424,6 +424,19 @@ function prepare
 
     onhost_cacheclouddata
     ${mkclouddriver}_do_prepare
+    if (( $want_ipv6 > 0 )); then
```

Comment:
why did you put this here rather than into the implementation of ${mkclouddriver}_do_prepare? the functionality is backend specific. IMHO this belongs into ./mkcloud-driver-libvirt.sh:function libvirt_do_prepare() to not break the backend physical or dirmaint driver.

Reply:
Well, I mean that's the interesting question, isn't it? I daresay radvd is backend agnostic, which is why it's in the generic function.

Reply:
So it's useful in dirmaint or physical environments? Why is this different to how ipv4 addresses are done?

Reply:
It will likely be required for physical environments, unless we're expecting non-libvirt environments to already provide some form of IPv6 RA infrastructure before we start?

Reply:
Well, I might be wrong, but I would assume a non-libvirt environment has already a IPv6 RA infrastructure and we shouldn't interfere with it. My natural tendency would be to move this into the libvirt backend case , as hinted already above. if you disagree then I would vote for a separate option for this (which defaults to off unless its libvirt)